essential-projects/data_model
# Feature/remove_iam_from_data_layer

## What did you change?

To have a clear separation of concerns I have removed all IAM-related functionality from the data layer.

Functionality that is no longer available after this PR is merged:
- Property-filtering on Pojos based on claims
- Read-/Write-Access checks based on claims

This PR also includes this PR on `data_model_contracts`: https://github.com/essential-projects/data_model_contracts/pull/12

## How can others test the changes?

You can just start the application with this feature branch and the `data_model_contracts` feature branch mentioned above included in the setup. You should still be able to execute processes and do everything as you like - as long as it doesn't require IAM-related functionality like property-filtering or read-/write-checks based on claims.

## PR-Checklist

Please check the boxes in this list after submitting your PR:
